test/packaging/guess-package-name.py

```diff
@@ -38,10 +38,14 @@
 python = TestSCons.python
 test = TestSCons.TestSCons()
 tar = test.detect('TAR', 'tar')
+bzip = test.where_is('bzip2')
```

Comment:
I don't think this entire test needs to be skipped if bzip2 isn't installed, only the bit around line 93

Reply:
In fact, since the Python stdlib tar module can handle .bz2 tarballs, there's no reason to skip for either tar or bzip2 not found as an external command, but that would require some recoding - different PR.